### PR TITLE
Add Tooltip

### PR DESCRIPTION
**Pull request by j-x-han**
Adds `Tooltip`, with inverted options. White (info) by default, black for action tooltip.

~~**Design Link:** https://chanzuckerberg.invisionapp.com/console/share/UWVLU4OH37F/511021232/play~~
**Design Link (updated):** https://www.figma.com/file/EaRifXLFs54XTjO1Mlszkg/Science-Design-System-Reference?node-id=642%3A4116

Default white info tooltip:
image

Inverted black action tooltip:
image

### CODE REVIEW

File: src/core/Tooltip/index.tsx
Comment (j-x-han) [inline src/core/Tooltip/index.tsx:28]: This corresponds to `font-body-xs`, but I'm not sure how to apply the mixin while using `makeStyles`.

I also tried applying it with emotion like this: 
```
export const StyledTooltip = styled(Tooltip)`
  &.MuiTooltip-tooltip {
    ${fontBody("xs")};
  }
`;
```
but couldn't get the style to apply. I'm pretty sure the class name is `.MuiTooltip-tooltip` based on the [docs](https://material-ui.com/api/tooltip/#css), so not sure why it's not showing up.

Any suggestions on how to clean this up are greatly appreciated! @tihuan

Comment (tihuan) [reply src/core/Tooltip/index.tsx:28]: Hurray! Thanks so much for the awesome PR, @j-x-han 🎉 🙏 !

I have been doing some research on this new use case and might have a solution 🤞 Will send out a PR against this one in a bit 😄 Thank you!

**Review by tihuan — APPROVED** [review]
LGTM after merging https://github.com/chanzuckerberg/sci-components/pull/22#pullrequestreview-653711427 🙏 Thanks again, @j-x-han 🚀 🚀 🚀 !

**Comment by j-x-han** [comment]
Refactored the name to just `Tooltip` and added the inverted options, PTAL!

**Review by tihuan — APPROVED** [review]
Looks amazing! Just one tiny change in this [PR](https://github.com/chanzuckerberg/sci-components/pull/23/files) 🙏 

PTAL again thank you!!

Once merged please kindly run `yarn build && npm version patch && npm publish` to publish your Tooltip to `npm` 🎉 

Thank you and congrats on shipping one more component for the team 🥳

**Pull request merged**
